common/config/rush/pnpmfile.js

```diff
@@ -36,5 +36,9 @@ function readPackage(packageJson, context) {
     packageJson.dependencies['ajv'] = '~6.12.5';
   }
 
+  if (packageJson.name === '@aws-sdk/middleware-retry') {
+    delete packageJson.dependencies['react-native-get-random-values'];
```

Comment:
Is this going to cause problems for people who install Rush?

Reply:
As far as I can tell the module is never loaded - either because the functionality that uses it is not exercised by Rush or because we are not on the react-native platform. All it does is add further dependencies, and peer dependencies, which do cause spurious warnings for the installer.